src/views/Accessibility/AccessibiltyRequest/Documents/index.tsx

```diff
@@ -50,18 +49,13 @@ const AccessibilityDocumentsList = ({
         Header: t('documentTable.header.actions'),
         Cell: ({ row }: any) => (
           <>
-            <UswdsLink
-              aria-describedby={`doc-link-${row.original.id}`}
-              target="_blank"
-              asCustom={Link}
-              to={row.original.url}
-            >
+            <a target="_blank" rel="noreferrer" href={row.original.url}>
```

Comment:
Should this be a real naked a tag, or should we still be using the USWDS link for styling? I honestly don't know...

Reply:
I'll use the component. Thanks for catching this!